uefi/src/lib.rs

```diff
@@ -101,6 +118,12 @@
 //!
 //! The full text of the license is available in the [license file][LICENSE].
 //!
+//! # Terminology (in this Crate)
+//!
+//! Although we sometimes use the terms "EFI" and "UEFI", both can be used
+//! interchangeably. We prefer "EFI" when talking about EFI images due to
+//! established naming conventions. Otherwise, we are using "UEFI".
```

Comment:
I'd rather just consistently use UEFI; the terms really are interchangeable at this point.